core/tests/protractor_utils/PreferencesPage.js

```diff
@@ -187,16 +187,18 @@ var PreferencesPage = function() {
 
   this.expectPreferredAudioLanguageToBe = async function(language) {
     await waitFor.visibilityOf(
-      selectedAudioLanguageElement,
-      'selectedAudioLanguageElement taking too long to appear.');
-    expect(await selectedAudioLanguageElement.getText()).toEqual(language);
+      audioLanguageSelector,
+      'audio language selector taking too long to appear.');
+    expect(await audioLanguageSelector.getText())
+      .toEqual(language);
```

Comment:
nit: Please move this to the previous line.

Reply:
done